ifermi/plot.py

```diff
@@ -1070,6 +1085,7 @@ def _get_plot_data(
         hide_slice: bool = False,
         hide_labels: bool = False,
         hide_cell: bool = False,
+        plot_index: List[int] = None
```

Comment:
You don't actually use this to select the FermiSlices to plot.

See my comments above about how this should be implemented.